### PR TITLE
doc: Add EKS node-init DaemonSet to mount BPF filesystem

### LINKED ISSUE #8163: add instructions to mount BPFFS to EKS guide
After following [the guide for installing on EKS](http://docs.cilium.io/en/v1.5/gettingstarted/k8s-install-eks/), Cilium logs contained the following:

```
2019-05-29T20:53:24.741595476Z level=warning msg="================================= WARNING ==========================================" subsys=bpf
2019-05-29T20:53:24.741600184Z level=warning msg="BPF filesystem is not mounted. This will lead to network disruption when Cilium pods" subsys=bpf
2019-05-29T20:53:24.741602404Z level=warning msg="are restarted. Ensure that the BPF filesystem is mounted in the host." subsys=bpf
2019-05-29T20:53:24.741604504Z level=warning msg="https://docs.cilium.io/en/stable/kubernetes/requirements/#mounted-bpf-filesystem" subsys=bpf
2019-05-29T20:53:24.741606702Z level=warning msg="====================================================================================" subsys=bpf
```

We should maybe link to the guide [here](http://docs.cilium.io/en/v1.5/kubernetes/requirements/?highlight=mount#mounted-bpf-filesystem).

### PR DESCRIPTION
**Pull request by tgraf**
Fixes: #8163

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/8173)

### CODE REVIEW

**Comment by tgraf** [comment]
test-docs-please

File: examples/kubernetes/node-init/eks-node-init.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                  mv /tmp/sys-fs-bpf.mount /etc/systemd/system/
+                  echo "Installed sys-fs-bpf.mount to /etc/systemd/system/"
+                elif [ -d "/lib/systemd/system/" ]; then
+                  mv /tmp/sys-fs-bpf.mount /lib/systemd/system/
+                  echo "Installed sys-fs-bpf.mount to /lib/systemd/system/"
+                fi
+
+                systemctl enable sys-fs-bpf.mount
+                systemctl start sys-fs-bpf.mount
+              EOF
```
Comment (ianvernon) [inline examples/kubernetes/node-init/eks-node-init.yaml]: Is this `EOF` supposed to be here?

Comment (tgraf) [reply examples/kubernetes/node-init/eks-node-init.yaml]: No, it is not. I just realized that the 2nd run to test this was ignored due to a check in the container image itself. Need to retest from scratch.

**Comment by tgraf** [comment]
@ianvernon Tested on a new fresh EKS cluster

**Comment by tgraf** [comment]
test-docs-please

**Pull request merged**